### PR TITLE
Added turn on lights for dark mode.

### PR DESCRIPTION
**Pull request by jorgeblacio**

### CODE REVIEW

File: iOS-Email-Client/Libs/AESCipher.swift [hunk 1]
```diff
@@ -55,7 +55,7 @@ class AESCipher {
         let options = CCOptions(kCCOptionPKCS7Padding)
         
         var cryptor: CCCryptorRef? = nil
-        let cryptorPointer = UnsafeMutablePointer<CCCryptorRef?>(&cryptor)
+        let cryptorPointer = UnsafeMutablePointer<CCCryptorRef?>( &cryptor)
```
Comment (Hirobreak) [inline iOS-Email-Client/Libs/AESCipher.swift:58]: ?

File: iOS-Email-Client/Views/EmailTableViewCell.swift [hunk 2]
```diff
@@ -67,6 +67,7 @@ class EmailTableViewCell: UITableViewCell{
     var email: Email!
     var emailState: Email.State!
     var isLoaded = false
+    var hasTurnedOnLights: Bool? = nil
```
Comment (Hirobreak) [inline iOS-Email-Client/Views/EmailTableViewCell.swift:70]: I think this should be handled in EmailState

File: iOS-Email-Client/Libs/AESCipher.swift (hunk 1 above)
Comment (jorgeblacio) [reply iOS-Email-Client/Libs/AESCipher.swift:58]: 🤷🏻‍♂️

File: iOS-Email-Client/Views/EmailTableViewCell.swift (hunk 2 above)
Comment (jorgeblacio) [reply iOS-Email-Client/Views/EmailTableViewCell.swift:70]: Ok, I was between those two, makes more sense to be part of the state.

**Review by Hirobreak — APPROVED** [review]

**Pull request merged**